admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -109,6 +109,112 @@ $product-page-padding-bottom: 80px !default;
   }
 
   // Components
+  .product-header-v2 {
+    display: flex;
+    flex-direction: row;
+    gap: 25px;
+    padding: 0.625rem;
+    margin-bottom: 1rem;
+    background: $white;
+    @include border-radius(0.375rem);
+    box-shadow: 0 0 12px rgba(0, 0, 0, 0.15);
+
+    > * {
+      flex-grow: 1;
+    }
+
+    $product-cover-size: 110px;
+
+    .product-header-cover {
+      max-width: $product-cover-size;
+
+      .image_preview-widget {
+        width: $product-cover-size;
+        height: $product-cover-size;
+
+        img {
```

Comment:
I would just use .img-fluid on the image, no need for these styles. Set width and height parameters on the image directly to improve CLS in modern browsers.

Reply:
Ok I removed that part, because you're right my image already used the `img-fluid` class However I still need to limit the size of its container But it's the only one that is limited now

Reply:
Yes, play with the width of the container, let everything inside adapt. :-)